x-pack/legacy/plugins/uptime/server/lib/adapters/monitor_states/elasticsearch_monitor_states_adapter.ts

```diff
@@ -4,607 +4,58 @@
-                            }
-                            if (check.container != null) {
-                              containerIds.add(check.container.id);
-                            }
-                            if (check.tls != null) {
-                              tls.add(check.tls);
-                            }
-                          }
-                        }
-                        
-                        // We just use the values so we can store these as nested docs
-                        result.checks = checks.values();
-                        result.put("@timestamp", maxTs);
-                        
-                        
-                        Map summary = new HashMap();
-                        summary.up = checks.entrySet().stream().filter(c -> c.getValue().monitor.status == "up").count();
-                        summary.down = checks.size() - summary.up;
-                        result.summary = summary;
-                        
-                        Map monitor = new HashMap();
-                        monitor.ip = ips;
-                        monitor.name = name;
-                        monitor.status = summary.down > 0 ? (summary.up > 0 ? "mixed": "down") : "up";
-                        result.monitor = monitor;
-                        
-                        Map observer = new HashMap();
-                        Map geo = new HashMap();
-                        observer.geo = geo;
-                        geo.name = geoNames;
-                        result.observer = observer;
-                        
-                        if (!podUids.isEmpty()) {
-                          result.kubernetes = new HashMap();
-                          result.kubernetes.pod = new HashMap();
-                          result.kubernetes.pod.uid = podUids;
-                        }
-      
-                        if (!containerIds.isEmpty()) {
-                          result.container = new HashMap();
-                          result.container.id = containerIds;
-                        }
-                        
-                        if (!tls.isEmpty()) {
-                          result.tls = new HashMap();
-                          result.tls = tls;
-                        }
-      
-                        return result;
-                        `,
-                },
-              },
-            },
-          },
-        },
-      },
-    };
-
-    const result = await this.database.search(request, params);
-    return { afterKey, result };
-  }
-
-  private getMonitorBuckets(queryResult: any, statusFilter?: any) {
-    let monitors = get(queryResult, 'aggregations.monitors.buckets', []);
-    if (statusFilter) {
-      monitors = monitors.filter(
-        (monitor: any) => get(monitor, 'state.value.monitor.status') === statusFilter
-      );
-    }
-    return monitors;
-  }
+    statusFilter?: string
+  ): Promise<GetMonitorStatesResult> {
+    const size = 10;
```

Comment:
Personally I'm not nuts about constants files just because in practice I rarely see the constants changed on any regular basis. They make the most sense IMHO when a number is shared between places in code in a way that must be coordinated.